### PR TITLE
chore: add pr & issue templates

### LINKED ISSUE #29: Add issue and PR templates
Maybe also add them for eludris/pilfer and eludris/pengin? 

also maybe maybe add dumbified versions for docs and elusite??? :lol:

### PR DESCRIPTION
**Pull request by toolifelesstocode**
This adds PR and Issue templates that will be used when one creates a bug report/feature request issue, or a pull request.

- Closes #29

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
+        - Effiw
+        - Effis
+        - Eludris
```
Comment (ooliver1) [inline .github/ISSUE_TEMPLATE/bug_report.yml:18]: I don't feel like these scopes make any sense. Effis/Effiw is part of Eludris, Effiw as in effiwbin? That should go to enoki's repo.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      options:
+        - label: Have you checked the documentation?
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproduction
+      description: Insert any code or steps to produce this issue.
```
Comment (ooliver1) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]:
```suggestion
      description: Insert any code or steps to reproduce this issue.
```

**Review by EnokiUN — APPROVED** [review]

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
+        - Effis
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]: effis is still a part of eludris, I'd recommend removing the `Eludris` option and adding ones for the rest of the microservices + todel

**Review by EnokiUN — CHANGES_REQUESTED** [review]

**Review by EnokiUN — APPROVED** [review]

**Comment by EnokiUN** [comment]
are you going to also pr this into pengin, pilfer and docs/elusite as discussed in the issue?

**Comment by toolifelesstocode** [comment]
> are you going to also pr this into pengin, pilfer and docs/elusite as discussed in the issue?

Yes

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      label: Checks
+      options:
+        - label: Have you checked the documentation?
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproduction
```
Comment (ooliver1) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]:
```suggestion
      label: Minimal Reproducible Code
```

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (EnokiUN) [reply .github/ISSUE_TEMPLATE/bug_report.yml:32]: Minimum Reproducible Sample/Code?

Comment (ooliver1) [reply .github/ISSUE_TEMPLATE/bug_report.yml:32]: ~~wawa~~ ok

**Review by ooliver1 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+name: Bug Report
+description: Report any bugs you encounter.
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]: Can we perhaps improve this to something like "Please enter the details of your bug" or the like

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      label: What is the issue?
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATE/bug_report.yml:15]: Maybe change this to `CI`?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        - Effis
+        - Todel
+        - Pandemonium
+        - Oprish
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATE/bug_report.yml:19]: Also maybe add a `CLI` field now that there is one

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+name: Feature Request
+description: Request a feature that you would like Eludris to support
```
Comment (EnokiUN) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: like the previous one, "Please enter the details of the feature you want added" or something along the lines

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+<!-- Link any issues if applicable.
+Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
+-->
+
+<!-- Uncomment based on the type of your changes below -->
+
+<!--
+## This is a **Code Change**
+
+- [ ] Docs have been updated if they need to be.
```
Comment (EnokiUN) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] Docs have been updated if they needed to be. 
```
grammar

**Review by EnokiUN — CHANGES_REQUESTED** [review]

**Comment by EnokiUN** [comment]
good pr 11/10 would review again

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+      options:
+        - label: Have you checked the documentation?
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproducible Sample/Code
+      description: Insert any code or steps to help reproduce this issue.
```
Comment (ooliver1) [inline .github/ISSUE_TEMPLATE/bug_report.yml:33]: If this has a `.`, shouldn't the other descriptions have it too?

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 10 above)
Comment (EnokiUN) [reply .github/ISSUE_TEMPLATE/bug_report.yml:33]: IMO remove them all

Comment (ooliver1) [reply .github/ISSUE_TEMPLATE/bug_report.yml:33]: IMO delete this PR

**Review by ooliver1 — CHANGES_REQUESTED** [review]

**Review by ooliver1 — CHANGES_REQUESTED** [review]
:confused:

**Review by EnokiUN — APPROVED** [review]

**Comment by EnokiUN** [comment]
LGTM. remember to yoink to other repos

**Review by ooliver1 — APPROVED** [review]
Nitpick, looks partially good to me :trollface:

**Comment by EnokiUN** [comment]
I'll be waiting for your other 4 prs, glhf

**Pull request merged**